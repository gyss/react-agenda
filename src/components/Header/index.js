import React from 'react';

import './styles.css';

function Header({ children }) {
    return (
      <div className="header">
        <h2>{ children }</h2>
      </div>
    );
}

export default Header;