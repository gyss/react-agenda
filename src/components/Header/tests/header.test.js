import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from '../index';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
});

it('renders correctly', () => {
  const tree = renderer.create(
    <Header>React Agenda 2017</Header>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});