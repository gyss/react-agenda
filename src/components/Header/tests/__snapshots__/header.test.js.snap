exports[`test renders correctly 1`] = `
<div
  className="header">
  <h2>
    React Agenda 2017
  </h2>
</div>
`;
